controller/src/main/java/org/jboss/as/controller/ProxyStepHandler.java

```diff
@@ -267,6 +276,20 @@ public void handleResult(OperationContext.ResultAction resultAction, OperationCo
         }
     }
 
+    private ModelNode processResponseHeaders(ModelNode responseHeaders) {
+        if (!responseHeaders.hasDefined(ACCESS_CONTROL) || !forServer) {
+            return responseHeaders;
+        } else {
+            ModelNode result = responseHeaders.clone();
+            for (ModelNode accItem : result.get(ACCESS_CONTROL).asList()) {
+                ModelNode itemAddrNode = accItem.get("absolute-address");
```

Comment:
I wouldn't say this is a blocker, but should `"absolute-address"` be a constant since it's used elsewhere as well? I don't think it's  a big deal either way.